### PR TITLE
Adds ability to add Comments to project

### PR DESCRIPTION
**Pull request by mateoclarke**
Closes https://github.com/cityofaustin/atd-data-tech/issues/4375

Includes UI & DB changes, use `moped-test` or `local` for testing.

To test, go to the "Comments" tab on any project and add comments 

https://user-images.githubusercontent.com/5697474/118898714-d1f3f080-b8d2-11eb-8c8d-079e95fe1e31.mov

### CODE REVIEW

File: moped-editor/src/App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {
   createHttpLink,
 } from "@apollo/client";
 
+var pckg = require("../package.json");
```
Comment (sergiogcx) [inline moped-editor/src/App.js:20]: Is this line needed. if not get rid.

File: moped-editor/src/App.js [hunk 2]
```diff
@@ -48,6 +49,8 @@ const App = () => {
     cache: new InMemoryCache(),
   });
 
+  console.log(`🛵 ${pckg.name} ${pckg.version} 🛵`);
```
Comment (sergiogcx) [inline moped-editor/src/App.js:52]: git rid! 🙏

**Review by sergiogcx — DISMISSED** [review]
It needs updating, see amenity's comments and conflicts below. Maybe merge main into this branch?

File: moped-editor/src/App.js (hunk 2 above)
Comment (mateoclarke) [reply moped-editor/src/App.js:52]: Sorry, I disagree. I want to keep this easily available until we decide where it should live in the app UI.

**Pull request closed without merging**